103163 Allocated Material

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files - Source/rm/b-rcptd.w
Source/rm/w-rcpt.w
Source/jc/dJobMat.w
Source/panels/p-allocated.w
Source/sharpshooter/b-job-mat-all.w
Source/sharpshooter/b-job-mat-last-all.w
Source/sharpshooter/w-jobAllocated.w

sewa67 committed:
103163 Allocated Material
Files - Source/rm/b-rcptd.w
Source/rm/w-rcpt.w
Source/jc/dJobMat.w
Source/panels/p-allocated.w
Source/sharpshooter/b-job-mat-all.w
Source/sharpshooter/b-job-mat-last-all.w
Source/sharpshooter/w-jobAllocated.w

sewa67 committed:
103163 Allocated Material
Files - Source/addon/rm/option.w
Source/addon/rm/rmtransa.w
Source/jc/dJobMat.w
Source/jc/JobProcs.p
Source/rm/b-rcptd.w
Source/rm/rmtransa.w
Source/rm/w-rcpt.w
Source/sharpshooter/b-job-mat-all.w
Source/sharpshooter/b-job-mat-last-all.w
Source/sharpshooter/w-jobAllocated.w
Source/panels/p-allocated.w

sachin448 committed:
103163 Allocated Material
File - Source/jc/dJobMat.w
Source/sharpshooter/b-job-mat-all.w

sewa67 committed:
103163 Allocated Material
Files - Source/jc/dJobMat.w
Source/sharpshooter/b-job-mat-all.w
Source/sharpshooter/b-job-mat-last-all.w
Source/sharpshooter/w-jobAllocated.w

sewa67 committed:
103163 Allocated Material
Files - Source/jc/jc-all2.p
Source/sharpshooter/b-job-mat-all.w
Source/sharpshooter/w-jobAllocated.w
Source/windows/l-item5.w

sewa67 committed:
103163 Allocated Material
Files - Source/jc/dJobMat.w
Source/windows/l-item5.w

sewa67 committed:
103163 Allocated Material
Files - Source/jc/JobProcs.p
Source/sharpshooter/w-jobAllocated.w
Source/windows/l-jobpo.w

sewa67 committed:
103163 Allocated Material
Files - Source/windows/l-jobpo.w

sewa67 committed:
103163 Allocated Material
Files - Source/jc/JobProcs.p
Source/sharpshooter/b-job-mat-all.w

ron-advantzware reviewed (changes requested):
jc/JobProcs.p - lines 215, 254 and 609 use ROWID to locate a record but also contain using company as part of the WHERE phrase.  When using ROWID, that is all the WHERE phrase requires.  Please remove company references.

jc/jc-all2.p - table referenced at line 20 does not have an explicit LOCK status (thus it's an implied SHARE-LOCK).  Please add the appropriate LOCK status EXCLUSIVE-LOCK and/or NO-LOCK

sharpshooter/b-job-mat-all.w - line 349 has an empty trigger that does absolutely nothing, please remove or modify to include intended code.

sewa67 committed:
103163 Allocated Material
Files - Source/jc/jc-all2.p
Source/jc/JobProcs.p
Source/sharpshooter/b-job-mat-all.w
